src/Guide/Views.hs

```diff
@@ -695,32 +703,33 @@ renderSearch mbSearchQuery =
 -- have to start invalidating 'CacheCategoryList' in more things in
 -- 'Cache.invalidateCache'.
 renderCategoryList :: MonadIO m => [Category] -> HtmlT m ()
-renderCategoryList allCats = cached CacheCategoryList $ do
-  div_ [id_ "categories"] $
-    for_ (groupWith (view group_) allCats) $ \catsInGroup ->
-      div_ [class_ "category-group"] $ do
-        -- Grandcategory name
-        h2_ $ toHtml (catsInGroup^?!_head.group_)
-        -- Finished categories
-        do let cats = filter ((== CategoryFinished) . view status) catsInGroup
-           unless (null cats) $
-             div_ [class_ "categories-finished"] $ do
-               mapM_ mkCategoryLink cats
-        -- In-progress categories, separated with commas
-        do let cats = filter ((== CategoryWIP) . view status) catsInGroup
-           unless (null cats) $
-             div_ [class_ "categories-wip"] $ do
-               h3_ "In progress"
-               p_ $ sequence_ $ intersperse ", " $
-                 map mkCategoryLink cats
-        -- Stub categories, separated with commas
-        do let cats = filter ((== CategoryStub) . view status) catsInGroup
-           unless (null cats) $
-             div_ [class_ "categories-stub"] $ do
-               h3_ "To be written"
-               p_ $ sequence_ $ intersperse ", " $
-                 map mkCategoryLink cats
+renderCategoryList allCats = toHtmlRaw =<< (liftIO $ renderBST gen)
```

Comment:
Hm, how come just removing `cached CacheCategoryList` didn't work in this case?

Reply:
No, a problem with the list of Category